README.md

```diff
@@ -76,8 +76,6 @@ In order to contribute to the project you may want some of the following install
 - [Go](https://golang.org/doc/install#install) - The Go programming
   language
 - [Fly CLI](https://github.com/concourse/fly) - Concourse CLI
-- [Virtualbox](https://www.virtualbox.org/) - Virtualization box
-- [Vagrant](https://www.vagrantup.com/) - Portable dev environment
```

Comment:
The only reason one would need these things is to deploy [concourse locally](https://github.com/cloudfoundry/garden-runc-release#running-the-tests-with-fly). I think the link is readme for deploying concourse is also incorrect. I think I am in favor or either removing both references and rely on folks having concourse deployed outside of this repo or leave VirtualBox and Vagrant and fix the link to point to valid guide.

Reply:
Fair points. I'll work on trying to get the documentation updated.